### ISSUE TITLE
eWallet_MobileApp-Utilities (Live Env) #132

### ISSUE THREAD

**Issue by farahmohamad** [TNGSB/eWallet#178]
Defect Description : System did not validate the blank fields and allow user to proceed with "Transaction PIN" page. No error message displayed

Test Description : To validate when only one field are left blank - account number left blank

User ID : Annemhd1710

Refer the attached POT for your perusal.
[Defect_Mobile App #132.xlsx](https://github.com/TNGSB/eWallet/files/605905/Defect_Mobile.App.132.xlsx)

**Comment by mayyeap** [comment]
Please check the latest version i told you yesterday and close this case

**Comment by mayyeap** [comment]
![5f984bae-5a33-45db-bba5-64e232b4977a](https://cloud.githubusercontent.com/assets/23113169/20549307/fe735354-b165-11e6-834f-8532fc0abefa.jpg)

**Comment by farahmohamad** [comment]
Hi May,

I have tested in IOS it works fine but for Android no.

Kindly check other than TM UniFi as Celcom and ASTRO still having the same issue.
FYI, I tested all with the new builds.

Thanks.
![screenshot_20161123-111758](https://cloud.githubusercontent.com/assets/23113211/20550255/bff3bca0-b16e-11e6-8ded-4b69e237fc60.png)
![screenshot_20161123-111748](https://cloud.githubusercontent.com/assets/23113211/20550272/df5b747a-b16e-11e6-827b-a04ee7932a46.png)
